optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/shift/ShiftRestServiceImpl.java

```diff
@@ -131,58 +114,24 @@ public Boolean removeShift(Integer tenantId, Long id) {
 
     @Override
     @Transactional
-    public List<Long> addShiftsFromTemplate(Integer tenantId,
-            String startDateString, String endDateString) throws Exception {
-        ShiftTemplate template = getTemplate(tenantId);
-
-        if (null == template) {
-            throw new IllegalStateException("You cannot add shifts if you don't have a template!");
+    public List<Long> addShiftsFromTemplate(Integer tenantId, Integer lengthInDays) {
+        Collection<ShiftTemplate> shiftTemplates = getTemplate(tenantId);
+
+        TenantConfiguration tenantConfiguration = entityManager.find(Tenant.class, tenantId).getConfiguration();
+        ShiftFileParser.ParserOut parserOutput = shiftFileParser.parse(tenantId, tenantConfiguration, null,
+                                                                       lengthInDays, shiftTemplates);
```

Comment:
Personally I don't think wrapped parameter lines are good for readability, but since our formatter does not enforce chopping them down or even leaving them inline, it's really a suggestion to change that :)

Reply:
Make that suggestion to the KIE Formatter team (pretty sure the formatter did that)

Reply:
Yeah, I think they decided to leave it to the developer choose what looks better..